pyfolio/tears.py

```diff
@@ -1528,16 +1528,16 @@ def create_perf_attrib_tear_sheet(returns,
           {'style': ['momentum', 'size', 'value', ...],
            'sector': ['technology', 'materials', ... ]}
     """
-    portfolio_exposures, perf_attrib_data = perf_attrib.perf_attrib(
+    portfolio_exposures, perf_attrib_data = ep.perf_attrib.perf_attrib(
         returns, positions, factor_returns, factor_loadings, transactions,
         pos_in_dollars=pos_in_dollars
     )
 
     display(Markdown("## Performance Relative to Common Risk Factors"))
 
     # aggregate perf attrib stats and show summary table
-    perf_attrib.show_perf_attrib_stats(returns, positions, factor_returns,
-                                       factor_loadings, transactions)
+    ep.perf_attrib.show_perf_attrib_stats(returns, positions, factor_returns,
```

Comment:
I think we only want to move the calculation logic into empyrical.  The plotting logic should still live in pyfolio. cc @richafrank @twiecki does that match your expectation? My general mental model of our project responsibility breakdown is:

- empyrical: Low-level shared risk calculations.
- pyfolio: High-level visualizations of historical portfolio data.
- zipline: Financial simulation, using empyrical definitions for evaluation, outputs feeding into pyfolio.

In particular, I think pyfolio is the only project in this list that should have a matplotlib dependency.

Reply:
Yep, that matches my mental model.

Reply:
gotcha, I can move this back